### PR TITLE
Marchenko Vladlen 203 group

### PR DESCRIPTION
**Pull request by VladilenMarchenko**

### CODE REVIEW

File: pz-1-4/index.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+          <path fill-rule="evenodd"
+                d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"/>
+        </svg>
+      </div>
+      <div id="profile-name">
+        Владлен Марченко
+      </div>
+    </div>
+
+    <div id="sidebar-links">
```
Comment (Zadvornyi) [inline pz-1-4/index.html:29]: It is navigation menu, You must use ul list or nav for navigation. It is SEO and sematic rule

File: pz-1-4/index.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+         viewBox="0 0 16 16">
+      <path
+        d="M2.146 2.854a.5.5 0 1 1 .708-.708L8 7.293l5.146-5.147a.5.5 0 0 1 .708.708L8.707 8l5.147 5.146a.5.5 0 0 1-.708.708L8 8.707l-5.146 5.147a.5.5 0 0 1-.708-.708L7.293 8 2.146 2.854Z"/>
+    </svg>
+  </div>
+</div>
+
+<div id="main">
+  <div id="header">
+    <button id="sidebar-open">
```
Comment (Zadvornyi) [inline pz-1-4/index.html:112]: use button for button

Comment (Zadvornyi) [reply pz-1-4/index.html:112]: not div

File: pz-1-4/index.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    <div class="tour-card">
+      <a href="./pages/page1.html">
+
+        <img class="tour-card-img" src="./img/1.webp" alt="">
+        <div class="tour-card-info">
+          <p>
+            Тур вихідного дня
+            "Гостинність Чорнобаївки"
+          </p>
+          <div class="tour-card-icons">
```
Comment (Zadvornyi) [inline pz-1-4/index.html:135]: It is not valid code, It is not correct pull block element to inline element

**Review by Zadvornyi — APPROVED** [review]
Screenshot 2023-09-13 at 19 37 40

File: pz-1-4/index.html (hunk 3 above)
Comment (VladilenMarchenko) [reply pz-1-4/index.html:135]: I didn't get this issue

**Pull request closed without merging**